server/api/event/event.model.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var mongoose = require('bluebird').promisifyAll(require('mongoose'));
+var Schema = mongoose.Schema;
+
+var EventSchema = new Schema({
+  title: String,
+  description: String,
+  course: {type: Schema.Types.ObjectId, ref: 'Course'},
+  submissions: [{type : Schema.Types.ObjectId, ref: 'Submission '}],
+  locations: [String], // TODO, what format should this be in?
+  times: [{
+      start: Date,
+      end: Date
+  }]
+});
```

Comment:
If an admin creates an event or there are multiple instructors or the instructors switch.  Just for general papertrails and record-keeping.